__tests__/app.test.js

```diff
@@ -313,6 +313,23 @@ describe("any methods: handles all bad paths", () => {
 describe("DELETE /api/comments/:comment_id", () => {
   test("204: should respond with message no content", () => {
```

Comment:
I would re-word this test to say it responds with a status code of 204! We aren't getting any message